src/EventStore.Plugins/Authentication/AuthenticationSchemes.cs

```diff
@@ -0,0 +1,7 @@
+namespace EventStore.Plugins.Authentication {
+	public enum AuthenticationSchemes {
```

Comment:
It might be better to make authentication schemes strings so that any new ones don't require recompiling and re-releasing the plugins package

Reply:
Thanks, I've made the changes